.circleci/config.templ.yml

```diff
@@ -510,14 +510,9 @@ jobs:
     <<: *machine_executor
     parallelism: 5
     steps:
-      - when:
-          condition:
-            matches: { pattern: "main", value: << pipeline.git.branch >> }
```

Comment:
Should we still scope when these tests run? Maybe `main` + any release branches?

Reply:
Sorry, this was just a test to verify those tests in the CI before merging my PR, I'll revert this change now